src/server/editorServices.ts

```diff
@@ -2880,9 +2947,41 @@ namespace ts.server {
                         // Ensure project is ready to check if it contains opened script info
                         updateProjectIfDirty(project);
                     }
+
                     defaultConfigProject = project;
-                    // Create ancestor configured project
-                    this.createAncestorProjects(info, defaultConfigProject);
+                    retainProjects = defaultConfigProject;
+
+                    // If this configured project doesnt contain script info but
+                    // it is solution with project references, try those project references
+                    if (!project.containsScriptInfo(info) && project.isSolution()) {
+                        forEachResolvedProjectReferenceProject(
+                            project,
+                            (child, childConfigFileName) => {
+                                updateProjectIfDirty(child);
+                                // Retain these projects
+                                if (!isArray(retainProjects)) {
+                                    retainProjects = [project as ConfiguredProject, child];
+                                }
+                                else {
+                                    retainProjects.push(child);
+                                }
+
+                                // If script info belongs to this child project, use this as default config project
+                                if (projectContainsInfoDirectly(child, info)) {
+                                    configFileName = childConfigFileName;
+                                    configFileErrors = child.getAllProjectErrors();
+                                    this.sendConfigFileDiagEvent(child, info.fileName);
+                                    return child;
+                                }
+                            },
+                            ProjectReferenceProjectLoadKind.FindCreateLoad,
+                            `Creating project referenced in solution ${project.projectName} to find possible configured project for ${info.fileName} to open`

```

Comment:
>Creating project referenced in solution ${project.projectName} to find possible configured project for ${info.fileName} to open [](start = 29, length = 127)

Does this message ever find its way into telemetry or error messages?

Reply:
Dont think so.. It is a part of event for `ProjectLoadStart` which helps display loading configured project..